lib/snowflex.ex

```diff
@@ -75,6 +75,8 @@ defmodule Snowflex do
   end
 
   defp process_results({:selected, headers, rows}) do
+    null_to_nil? = Application.get_env(:snowflex, :map_nulls_to_nil?, true)
```

Comment:
Should this default to false to preserve current behavior and not introduce a breaking change?

Reply:
yes, it should